src/SDKs/ApplicationInsights/ApplicationInsights.Tests/ApplicationInsights.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>ApplicationInsights.Tests</PackageId>
+    <Description>ApplicationInsights.Tests Class library</Description>
+    <PackageId>ApplicationInsights.Tests</PackageId>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <TargetFrameworks>net452;</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="BasicTests\**" />
```

Comment:
@frankguodongchen what is the purpose of this ItemGroup?
Please remove as all files/directories are by default included in the project

Reply:
OK, updated.